actioncable/app/javascript/action_cable/index.js

```diff
@@ -1,58 +1,45 @@
+import adapters from "./adapters"
 import Connection from "./connection"
 import ConnectionMonitor from "./connection_monitor"
 import Consumer from "./consumer"
 import INTERNAL from "./internal"
+import * as logger from "./logger"
```

Comment:
`logger` stands out as inconsistent here. I think it should export a default object instead of individual functions to bring it in line:
```js
export default {
  log(…) {
    // …
  },

  enable() {
    // …
  },

  disable() {
    // …
  }
}
```

Reply:
Ok. The only reason I had them as separately exported functions instead of exporting a default object was to make it more tree-shakeable (since `enable` and `disable` wouldn't be automatically pulled in everywhere that uses the `log` method). But if we're going to put it all under one object, I think we can go with the `logger.enabled = true|false` API instead of having `enable()`/`disable()` methods. I'll go that route since you said that API was also fine.

Reply:
Refactored in https://github.com/rmacklin/rails/commit/5819e582aa75fa5beb5772fd5a92e732b927261c